lib/react_on_rails/helper.rb

```
@@ -149,6 +149,17 @@
       rendering_fiber.resume
     end
 
+    def rsc_react_component(component_name, options = {})
+      rendering_fiber = Fiber.new do
+        res = internal_rsc_react_component(component_name, options)
+        res.each_chunk do |chunk|
+          Fiber.yield chunk
+        end
+        Fiber.yield nil
+      end
```

rubocop

[convention] Redundant assignment before returning detected.

(Style/RedundantAssignment)